.github/workflows/release.yml

```diff
@@ -51,6 +50,5 @@ jobs:
         with:
           files: ./bin/*
           body_path: ./CHANGELOG.md
-          prerelease: "true"
```

Comment:
I would avoid performing an official release from the CI, I think we should keep it as prerelease until agree on officially releasing it.

In any case updating a prerelease to a release is just 3 clicks away :)

Reply:
The problem here is that I manually created a release and the CI job changed it to prerelease which I kind of did not like.

And we need real releases, because with prerelease, "latest" download does not work and we're downloading latest version of the client from the installer.

Reply:
you are saying that the job edited a previously created release? that sounds like an issue with the job. I do not remember this happening when I worked on it at fuseml/fuseml

Reply:
I don't think this is such a big deal, we can always delete a release and it's associated tag (which I recommend that we do anyway with all these "tests" that we've been making, before releasing the official MVP version).